From cbecdbe5b14c08240aaf12cdb8cec010cab6ad09 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Vin=C3=ADcius=20Camillo?=
 <98857171+vrcamillo@users.noreply.github.com>
Date: Wed, 22 Mar 2023 12:19:50 -0300
Subject: [PATCH] Upgrading LibPoly. (#9580)

Upgrading LibPoly to fix a segmentation violation.

Fixes #9567.
---
 cmake/FindPoly.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/FindPoly.cmake b/cmake/FindPoly.cmake
index 9041890a203..f28b5a005e9 100644
--- a/cmake/FindPoly.cmake
+++ b/cmake/FindPoly.cmake
@@ -45,7 +45,7 @@ if(NOT Poly_FOUND_SYSTEM)

   include(ExternalProject)

-  set(Poly_VERSION "1383809f2aa5005ef20110fec84b66959518f697")
+  set(Poly_VERSION "126147f1ceae9f771a68bad9cbc199cf96daec46")

   check_if_cross_compiling(CCWIN "Windows" "")
   if(CCWIN)
@@ -158,7 +158,7 @@ if(NOT Poly_FOUND_SYSTEM)
     Poly-EP
     ${COMMON_EP_CONFIG}
     URL https://github.com/SRI-CSL/libpoly/archive/${Poly_VERSION}.tar.gz
-    URL_HASH SHA1=e3da80491b378a4d874073d201406eb011f47c19
+    URL_HASH SHA1=5ebbd00c8dc8b731b5382701a23c60607bcb2804
     PATCH_COMMAND
       sed -i.orig
       "s,add_subdirectory(test/polyxx),add_subdirectory(test/polyxx EXCLUDE_FROM_ALL),g"
